tools/integrations/manage.sh

```diff
@@ -135,39 +114,121 @@ cmd_bump_version() {
     *)
       echo "Invalid bump_version option: $bump_version. Valid options are major, minor, patch"
       exit 1
+      ;;
   esac
 
   bumped_version="$major_version.$minor_version.$patch_version"
-  # This image should not already exist, if it does, something weird happened
-  _error_if_tag_exists "$image_name:$bumped_version"
-  echo "$connector:$current_version will be bumped to $connector:$bumped_version"
+  if [[ "$FORCE_BUMP" == "true" ]] || [[ "$bumped_version" != "$master_version" && "$bumped_version" != "$branch_version" ]]; then
+    _error_if_tag_exists "$image_name:$bumped_version"
+    echo "$connector:$branch_version will be bumped to $connector:$bumped_version"
+
+    # Set outputs back to Github Actions for later steps
+    echo ::set-output name=master_version::"${master_version}"
+    echo ::set-output name=bumped_version::"${bumped_version}"
+    echo ::set-output name=bumped::"true"
```

Comment:
This GHA output is important, its what it used to tell downstream steps whether or not a bump occurred

Reply:
Reason this is better is for future runs when a bump has already occurred, future GHA runs will not fail or run unnecessarily